bokehjs/test/unit/models/expressions/cumsum.ts

```diff
@@ -10,11 +10,11 @@ describe("CumSum", () => {
 
     const s0 = new CumSum({field: 'foo'})
     const ret0 = s0.v_compute(source)
-    expect(ret0).to.be.equal(new Float64Array([1, 3, 6, 10]))
+    expect(ret0).to.be.equal(new Float32Array([1, 3, 6, 10]))
```

Comment:
In fact everywhere where code produces number arrays, `NumberArray` type should be used, to indicate the intent. I know, that the original changes weren't consistent either, but as you already are changing all this, let's do this properly.

Reply:
I'm fine with that though I assumed the intent was also to ensure that `NumberArray` was also in fact `Float32Array`. But I will go through and update ones that  return NumberArrray specifically.